src/cpp/r/session/graphics/RGraphicsDevice.cpp

```diff
@@ -671,6 +670,11 @@ void onBeforeExecute()
 
 } // anonymous namespace
     
+void playDisplayList()
```

Comment:
It gets called here: https://github.com/rstudio/rstudio/pull/684/files#diff-b68525afe6ff42cef3f71a8793d3af53R1369